### PR TITLE
View 2D: Color range slider 

### PR DESCRIPTION
**Pull request by PaulHax**
image

Related #133

### CODE REVIEW

**Review by thewtex — COMMENTED** [review]
Niiice!

I like the interface and tooltips. To the line above the slider, could we similarly also add smaller filled circles and hover tooltips above (maybe then put the slider tooltips below) that communicate that this is the full range and what its values are?

**Comment by PaulHax** [comment]
That line is a weekness.  How about this:
image

**Comment by thewtex** [comment]
> That line is a weekness. How about this:

Beautiful! :+1: :100:

File: packages/element/src/view-2d-controls-controller.ts [hunk 1]
```diff
@@ -75,4 +89,77 @@ export class View2dControls implements ReactiveController {
     const scale = Number(target.value);
     this.actor!.send({ type: 'setScale', scale });
   }
+
+  setTransferFunctionContainer(container: Element | undefined) {
+    if (container) {
+      this.transferFunctionEditor = new TransferFunctionEditor(container);
+
+      const colorTransferFunction = (function () {
```
Comment (thewtex) [inline packages/element/src/view-2d-controls-controller.ts:97]: -> class?

**Review by thewtex — APPROVED** [review]
Good to go, refactoring note here or in follow-up inline

File: packages/element/src/view-2d-controls-controller.ts (hunk 1 above)
Comment (PaulHax) [reply packages/element/src/view-2d-controls-controller.ts]: Yes, added ColorTransferFunction class.

**Pull request merged**